synapse/util/wheel_timer.py

```diff
@@ -91,7 +91,4 @@ def fetch(self, now):
         return ret
 
     def __len__(self):
-        l = 0
-        for entry in self.entries:
-            l += len(entry.queue)
-        return l
```

Comment:
What lint is picking this up?

Reply:
https://github.com/matrix-org/synapse/pull/2568/commits/0be99858f34165258af5d2865818f5baee35118b

"E741 says "do not use variables named ‘l’, ‘O’, or ‘I’"